content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md

```diff
@@ -0,0 +1,124 @@
+---
+title:  OCP/OKD Support
+weight: 4
+---
+
+To deploy Longhorn on a cluster provisioned with OpenShift 4.x, some additional configurations is required.
+
+- [Prerequisite](#prerequisite)
+  - [Add An Extra Disk to Longhorn Storage](#add-an-extra-disk-to-longhorn-storage)
+    - [Create Filesystem For The Device](#create-filesystem-for-the-device)
+    - [Mounting The Device On Boot with MachineConfig CRD](#mounting-the-device-on-boot-with-machineconfig-crd)
+    - [Label and Annotate The Node](#label-and-annotate-the-node)
+- [Install Longhorn](#install-longhorn)
+- [Upgrade Longhorn](#upgrade-longhorn)
+- [Uninstall Longhorn](#uninstall-longhorn)
+- [Reference](#reference)
+- [Main Contributor](#main-contributor)
+
+## Prerequisite
+
+If you don't want to use the default disk as the Longhorn storage, set ***defaultSettings.createDefaultDiskLabeledNodes*** true when installing Longhorn by helm:
```

Comment:
Actually, users don't need to set up a dedicated disk if they just try this setup, so this is not really a prerequisite but rather an example to teach users how to create a customized default longhorn disk by using `MachineConfig`. 

Thus, I would suggest change `prerequisite` to `Prepare a customized default longhorn disk (Optional)`, then move all original subsections into this section. At the beginning of the section, you can briefly explain the purpose of it and it's actually optional if users just use the default longhorn disk.

Reply:
Moved the section `prerequisite` to `Prepare a customized default longhorn disk (Optional)` and add more descriptions in this section. Thanks